src/main/webapp/app/exercises/programming/manage/build-plan-editor.component.ts

```diff
@@ -67,10 +74,16 @@ export class BuildPlanEditorComponent implements AfterViewInit, OnInit {
      */
     private loadBuildPlan(exerciseId: number) {
         this.isLoading = true;
-        this.buildPlanService.getBuildPlan(exerciseId).subscribe((buildPlan) => {
-            this.buildPlan = buildPlan.body!;
-            this.initEditor();
-            this.isLoading = false;
+        this.buildPlanService.getBuildPlan(exerciseId).subscribe({
+            next: (buildPlan) => {
+                this.buildPlan = buildPlan.body!;
+                this.initEditor();
+                this.isLoading = false;
+            },
+            error: () => {
+                this.alertService.error('artemisApp.programmingExercise.buildPlanFetchError');
```

Comment:
ToDo: test case that alertservice was called on error and loading=false afterwards